vendor-extra/LiberoPageBundle/src/EventListener/BuildView/ItemListListener.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\EventListener\BuildView;
+
+use FluentDOM\DOM\Element;
+use FluentDOM\DOM\Node\NonDocumentTypeChildNode;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use function array_map;
+use function iterator_to_array;
+use function Libero\ViewsBundle\array_has_key;
+
+final class ItemListListener
+{
+    use SimplifiedViewConverterListener;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function handle(Element $object, View $view) : View
+    {
+        return $view->withArgument(
+            'list',
+            [
+                'items' => array_map(
+                    function (NonDocumentTypeChildNode $child) use ($view) : array {
+                        return [
+                            'content' => $this->converter->convert(
+                                $child,
+                                '@LiberoPatterns/teaser.html.twig',
+                                $view->getContext()
+                            )
```

Comment:
Ref https://github.com/libero/browser/pull/64#issuecomment-479404044:

```suggestion
                            )->getArguments()
```

The converter returns the complete `View`, but the list template already includes the inclusion of the `teaser` template https://github.com/libero/browser/blob/029a20136526946ce13e2a7b01549368191ea4f7/vendor-extra/LiberoPatternsBundle/src/Resources/views/teaser-list.html.twig#L37 so the `teaser` template was ending up with the `View` rather than the arguments for it.